examples/deployments/terraform-gcp-github-actions/gce.tf

```diff
@@ -0,0 +1,53 @@
+resource "google_compute_instance" "chroma-db-server-instance-dev" {
+  name    = "chroma-db-server-instance-dev"
+  project = var.project
+  zone    = "europe-north1-c"
+
+  machine_type = "e2-small"
+
+  boot_disk {
+    initialize_params {
+      image = module.gce-container.source_image
+    }
+  }
+
+  attached_disk {
+    source      = google_compute_disk.pd.self_link
+    device_name = "data-disk-0"
+    mode        = "READ_WRITE"
+  }
+
+  metadata = {
+    gce-container-declaration    = module.gce-container.metadata_value
+    google-logging-enabled       = true
+    google-logging-use-fluentbit = true
+  }
+
+
+  labels = {
```

Comment:
Can labels also be attached to other resources? e.g. volumes, containers?

Reply:
Yeah they can, this specific label makes sure the OS is running the container 👍